supervisor/bootstrap.py

```diff
@@ -301,13 +301,19 @@ def filter_data(event, hint):
                     "channel": coresys.updater.channel,
                     "supervisor": coresys.supervisor.version,
                     "os": coresys.hassos.version,
+                    "host": coresys.host.info.operating_system,
+                    "kernel": coresys.host.info.kernel,
                     "core": coresys.homeassistant.version,
                     "audio": coresys.plugins.audio.version,
                     "dns": coresys.plugins.dns.version,
                     "multicast": coresys.plugins.multicast.version,
                     "cli": coresys.plugins.cli.version,
                 },
             )
+            scope.set_tag(
+                "installation_type",
+                f"{'generic' if coresys.hassos.available else 'hassos'}",
```

Comment:
Should we use the official names instead?
"Home Assistant OS", "Home Assistant Supervised"?

Reply:
it's more about sorting inside sentry (short is better because otherwise OS/Supervised is cut)

Reply:
we can use the ADR number?